playgrounds/sandbox/run.js

```diff
@@ -49,7 +50,7 @@ for (const generate of ['client', 'server']) {
 		}
 
 		const compiled = compile(source, {
-			dev: true,
+			dev: false,
```

Comment:
I don't think we want this?

Reply:
depends what we're testing — in this case, I needed to check that empty style declarations were being removed. we'll inevitably need to flip it back and forth occasionally. i'll revert this change for now though

```suggestion
			dev: true,
```